db/migrate/20200616160640_create_experience_points.rb

```diff
@@ -0,0 +1,10 @@
+class CreateExperiencePoints < ActiveRecord::Migration[5.1]
```

Comment:
I find calling `experience_points.exp` not ideal but I'm not sure what is the ideal alternative.

`experience.points` maybe be better, but calling the entity just Experience doesn't seem ideal either.
`experience_points.total` may be an alternative :thinking: 

We could think a bit ahead and use this entity to store the `streaks` as well, and make it more than it is now. With that in mind maybe we could come up with better names, like ¿stats?.

Reply:
I was thinking about the name as well because `experience_points.exp` is not very good.

We have a couple other entities which represent a many-to-many users and organizations relationship, but they were created to support a third entity: an exercise in `assignments`, a parent content in `indicators`, a discussion in... `discussions`.

I think we should strive to come up with a new domain name for this, which conveys "this user in this organization". I'm not a fan of `stats` because it's such a broad term.

A word that represents `one who follows a path` would be my most preferred choice (we have mumuki.io/paths, don't we?) but I'm not high enough to choose `Traveller`, `Explorer` or `Voyager`. We're not that dangerous either.

![image](https://user-images.githubusercontent.com/11304439/86407336-3d1ce000-bc8b-11ea-8383-341e9afce486.png)